### DISCUSSION TITLE
Can custom scripting possible in microcks to implement some business logics

### DISCUSSION THREAD

**Discussion by Athira31** [microcks/microcks#988] [Q&A] [answered]
In microcks can add functionalities to add two numbers and map it to response

**Comment by lbroudoux** [comment, marked as answer]
Yes sure!

You may want to look at [`SCRIPT` dispatcher](https://microcks.io/documentation/using/advanced/dispatching/#script-dispatcher) to implement business logic and then put the result into `requestContext`. Then you can use [request context expression](https://microcks.io/documentation/using/advanced/templates/#context-expression) when expressing your response using a template. That way you may re-inject the result of business logic computation into the response.

Hope it helps!

**Reply by lbroudoux** [reply]
Hey @Athira31! Did you have the opportunity to check ^^?
